docs/src/validator/overview/what-is-a-validator.md

```diff
@@ -0,0 +1,39 @@
+---
+title: What is a Validator?
+---
+
+A validator is a computer that helps to run the Solana network. Each validator executes a program that keeps track of all accounts on the Solana cluster and validates transactions being added to the network. Without validators, Solana would not be able to function.
+
+The more independent entities that run validators, the less vulnerable the cluster is to an attack or catastrophe that affects the cluster.
+
+> At [solanabeach.io](https://solanabeach.io/), you can see the location of all validators in the mainnet beta cluster. You can also see statistics and information on each validator at [solanabeach.io/validators](https://solanabeach.io/validators).
```

Comment:
Do more people use SolanaBeach or Validators.app to get general info on the validators? Is it worth mentioning both here? Or updating?

Reply:
Good catch.  I replaced this with a reference to the validator health report which is more appropriate.